pyanaconda/install.py

```diff
@@ -131,6 +134,10 @@ def doConfiguration(storage, payload, ksdata, instClass):
     # write out the user interaction config file
     screen_access.sam.write_out_config_file()
 
+    if ksdata.snapshot:
+        with progress_report(N_("Creating snapshots")):
+            ksdata.snapshot.execute(storage, ksdata, instClass)
+
```

Comment:
This (and the above ``step_count`` incrementation) should only happen when creating a ``post-install`` snapshot, right? I believe this should be fixed up in the next commit.